www/include/monitoring/status/ServicesHostGroups/serviceGridByHG.php

```diff
@@ -101,6 +103,23 @@
 
 $form = new HTML_QuickFormCustom('select_form', 'GET', "?p=" . $p);
 
+//adding hostgroup's select2 list
+$hostgroupsRoute = './include/common/webServices/rest/internal.php?object=centreon_configuration_hostgroup&action=list';
```

Comment:
use ./api/internal.php instead of ./include/common/webServices/rest/internal.php